pyresample/test/test_kd_tree.py

```diff
@@ -881,13 +899,16 @@ def test_nearest_swath_2d_mask_to_area_1n(self):
     def test_nearest_area_2d_to_area_1n(self):
         """Test 2D area definition to 2D area definition; 1 neighbor."""
         from pyresample.kd_tree import XArrayResamplerNN
+        from pyresample.test.utils import CustomScheduler
         import xarray as xr
         import dask.array as da
+        import dask
         data = self.data_2d
         resampler = XArrayResamplerNN(self.src_area_2d, self.area_def,
                                       radius_of_influence=50000,
                                       neighbours=1)
-        ninfo = resampler.get_neighbour_info()
+        with dask.config.set(scheduler=CustomScheduler(0)):
```

Comment:
This CustomScheduler is currently in pyresample, satpy, and I use it in Polar2Grid. It might also be copied to trollimage and maybe another library. I don't disagree and I've considered this context manager too, but I was worried it would distract from what is actually happening (we are configuring dask). Perhaps in a separate PR in the future? :crossed_fingers:

Reply:
Then it could be called `configure_dask_to_make_sure_we_do_not_compute_anything`?

Reply:
Let me see what I can do.

Reply:
Done. I named it `assert_max_dask_computes`.